src/System.Collections/src/System/Collections/Generic/SortedSet.TreeSubSet.cs

```diff
@@ -45,12 +45,6 @@ public TreeSubSet(SortedSet<T> Underlying, T Min, T Max, bool lowerBoundActive,
                 VersionCheckImpl();
             }
 
-            private TreeSubSet(SerializationInfo info, StreamingContext context)
```

Comment:
There are a ton of internal types like TreeSubSet (various KeyCollections, ValueCollections, Synchronized* etc). Do you think they all belong? Is TreeSubSet special in some way?